lib/stale.js

```diff
@@ -80,7 +80,68 @@ module.exports = class Stale {
 
     const query = queryParts.join(' ')
     const days = this.getConfigValue(type, 'days') || this.getConfigValue(type, 'daysUntilStale')
-    return this.search(type, days, query)
+    const results = await this.search(type, days, query)
+
+    const pinned = await this.getPinnedNumbers(type)
+    const nonPinned = results.data.filter(issue => pinned.includes(issue.number))
+    return {
+      ...results,
+      data: {
+        ...results.data,
+        total_count: nonPinned.length,
+        items: nonPinned,
+      }
+    }
+  }
+
+  async getPinnedNumbers(type) {
```

Comment:
We could potentially add a config setting to control if pinned issues should be ignored or not, but I wasn't sure if it was worth it.